extensions/resource-deployment/src/ui/resourceTypePickerDialog.ts

```diff
@@ -476,4 +614,18 @@ export class ResourceTypePickerDialog extends DialogBase {
 			this._installationInProgress = false;
 		}
 	}
+
+	private getAllResourceTags(resourceTypes: ResourceType[]): string[] {
```

Comment:
you are always using this._resourceTypes as the parameter value, is the parameter really needed?

Reply:
will get rid of this as well